draft-ietf-quic-transport.md

```diff
@@ -1728,9 +1728,9 @@ disable_migration (0x0009):
 
 max_ack_delay (0x000c):
 
-: An 8 bit unsigned integer value indicating the maximum amount of time in
-  milliseconds by which it will delay sending of acknowledgments.  If this
-  value is absent, a default of 25 milliseconds is assumed.
+: An 8-bit unsigned integer value to use as the maximum delay (in milliseconds)
```

Comment:
The intent of this transport param is for the receiver to communicate it's max ack delay to the sender, so I don't think the change from "indicating" to "to use as" is quite right.

How about "An 8-bit unsigned integer value indicating the maximum amount of time in milliseconds a receiver will delay sending an acknowledgement.  The receiver sends this value to the sender to set timeouts more accurately."?

I remember this was reworded a few times when I was writing it,

Reply:
Oh, that's right.  In this case, I how about "used as?"

> The maximum delay (in milliseconds) used to send acknowledgements encoded as an 8-bit unsigned integer.

Alternatively, I can close this PR and you can improve it as you see fit, @ianswett.